src/classes/User.js

```diff
@@ -0,0 +1,41 @@
+class User {
+  constructor(userData = {}) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.pantry = userData.pantry;
+    this.recipesToCook = [];
+  }
+
+  addRecipesToCook(recipe) {
+    this.recipesToCook.push(recipe);
+  }
+  
+  removeRecipesToCook(recipe) {
+    let recipeToRemove = this.recipesToCook.indexOf(recipe);
+    this.recipesToCook.splice(recipeToRemove, 1);
+    return this.recipesToCook;
+  }
+  filterRecipeByTag(tag) {
+    let filteredRecipesByTag = this.recipesToCook.filter((recipe) => {
+      if (recipe.tags.includes(tag)) {
+        return true;
+      }
+    });
+    return filteredRecipesByTag;
+  }
+  userFilteredRecipesByName(recipeName) {
```

Comment:
interesting name for this...could you write a bit about why you chose to specify `user` in the name?

Reply:
I would have gone for `filterRecipesByName()` to match line 18

Reply:
This was changed to reflect your suggestion and it now matches line 18.